docs/OV_Runtime_UG/deployment/deployment_intro.md

```diff
@@ -11,58 +11,44 @@
 
 @endsphinxdirective
 
-Once the [OpenVINO application development](../integrate_with_your_application.md) has been finished, usually application developers need to deploy their applications to end users. There are several ways how to achieve that:
+Once the [OpenVINO™ application development](../integrate_with_your_application.md) has been finished, application developers usually need to deploy their applications to end users. There are several ways to achieve that:
 
-- Set a dependency on existing prebuilt packages (so called _centralized distribution_):
-    - Using Debian / RPM packages, a recommended way for a family of Linux operation systems
-    - Using pip package manager on PyPi, default approach for Python-based applications
-    - Using Docker images. If the application should be deployed as a Docker image, developer can use a pre-built runtime OpenVINO Docker image as a base image in the Dockerfile for the application container image. You can find more info about available OpenVINO Docker images in the Install Guides for [Linux](../../install_guides/installing-openvino-docker-linux.md) and [Windows](../../install_guides/installing-openvino-docker-windows.md). 
-Also, if you need to customize OpenVINO Docker image, you can use [Docker CI Framework](https://github.com/openvinotoolkit/docker_ci) to generate a Dockerfile and built it. 
-- Grab a necessary functionality of OpenVINO together with your application (so-called _local distribution_):
-    - Using [OpenVINO Deployment manager](deployment-manager-tool.md) providing a convinient way create a distribution package
-    - Using advanced [Local distribution](local-distribution.md) approach
-    - Using [static version of OpenVINO Runtime linked into the final app](https://github.com/openvinotoolkit/openvino/wiki/StaticLibraries)
+- Set a dependency on the existing prebuilt packages, also called "centralized distribution":
+    - using Debian / RPM packages - a recommended way for Linux operating systems;
+    - using PIP package manager on PyPI - the default approach for Python-based applications;
+    - using Docker images - if the application should be deployed as a Docker image, use a pre-built OpenVINO™ Runtime Docker image as a base image in the Dockerfile for the application container image. For more information about OpenVINO Docker images, refer to the installation guides [for Linux from Docker](../../install_guides/installing-openvino-docker-linux.md) and [for Windows from Docker](../../install_guides/installing-openvino-docker-windows.md). 
```

Comment:
```suggestion
    - using Docker images - if the application should be deployed as a Docker image, use a pre-built OpenVINO™ Runtime Docker image as a base image in the Dockerfile for the application container image. For more information about OpenVINO Docker images, refer to [Installing OpenVINO on Linux from Docker](../../install_guides/installing-openvino-docker-linux.md) and [Installing OpenVINO on Windows from Docker](../../install_guides/installing-openvino-docker-windows.md). 
```
Sorry for this back and forth change. I think the best solution might be spelling out the full book name.